src/main/migrations/20180508093508_init.js

```diff
@@ -8,14 +8,20 @@ exports.up = function(knex, Promise) {
 				.references('wallets.id');
 			table.string('title');
 			table.string('content');
-			table.integer('createdAt').notNullable();
+			table
```

Comment:
We can't change this file now that people have run the wallet.

Create a new migration to make these schema changes using the `knex` CLI.